packages/jest-config/src/Defaults.ts

```diff
@@ -43,7 +43,6 @@ const defaultOptions: Config.DefaultOptions = {
   injectGlobals: true,
   listTests: false,
   maxConcurrency: 5,
-  maxWorkers: '50%',
```

Comment:
removed this from the default because  we are anyway calculating default values in [getMaxWorkers](https://github.com/k-rajat19/jest/blob/main/packages/jest-config/src/getMaxWorkers.ts#L38)